CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# v7.0.0 (2025-04-18)
+
+## OS Changes
+* Fix aws-signing-helper and IAM Roles Anywhere ([#451])
+* Clear configuration-files and services in migrator ([#456])
+* Drop shimpei and oci-add-hooks packages ([#458])
+* Restrict kernel dump collection to x86_64 ([#465])
+* Third party package updates ([#469], [#472])
+
+## Orchestrator Changes
+### Kubernetes
+* Apply upstream patches for EKS 1.25-1.27 ([#472])
```

Comment:
In early releases of Bottlerocket, we use to just include all the package updates in one single line. There have been a couple releases where we do a mix of a couple of things:
- List k8s updates individually
- Add them as part of the third-party package updates
- Use "third party" or "third-party"
- List individual packages, and keep a line for "third-party" package updates

We should revisit what's the format to follow, and be consistent again.